scripts/rest_examples/pubkeys-list.http

```diff
@@ -1,5 +1,5 @@
 // @no-log
-GET http://{{hostname}}:{{port}}/{{prefix}}/pubkeys/ HTTP/1.1
+GET http://{{hostname}}:{{port}}/{{prefix}}/pubkeys?limit=&offset= HTTP/1.1
```

Comment:
Maybe let's leave here some valid values? This way, it produces error, as the limit and offset are both empty.

Reply:
It does not produce an error, default values are set in the middleware if limit or offset are empty.

Reply:
Yeah I am sorry, I meant a warning saying that the values are empty.
Fair enough, I do not think it is that important, so np, just leave it as it is. I would just expect valid values here, but we can leave it as it is:) :+1: Looks good.